src/hooks/Orderbook/useDrawOrderbook.ts

```diff
@@ -96,228 +95,286 @@ export const useDrawOrderbook = ({
 
       setCanvasWidth(canvas.width / currentRatio);
       setCanvasHeight(canvas.height / currentRatio);
+    },
+    [canvas]
+  );
+
+  // Handle resize, sync to state
+  useEffect(() => {
+    const resizeObserver = new ResizeObserver((entries) => {
```

Comment:
one last thing - do we support IE? Looks like even latest internet explorer doesn't support this.

Reply:
checked in [slack](https://dydx-team.slack.com/archives/C03UV6K4KFY/p1722871900127219?thread_ts=1722871230.876329&cid=C03UV6K4KFY) tl;dr: nope (because you can't even connect a wallet on IE). longer answer: stats show that none of our users use IE. Checked ResizeObserver on [caniuse](https://caniuse.com/resizeobserver) and seems like we should be safe outside of IE and OperaMini (the latter I think is fine since it's supported in Opera)